raiden_contracts/tests/test_deploy_script.py

```diff
@@ -624,27 +624,34 @@ def test_deploy_token_with_balance(get_accounts, get_private_key):
                 mock_deployer.assert_called_once()
 
 
-def deploy_raiden_arguments(privkey: str, save_info: Optional[bool]) -> List:
+def deploy_raiden_arguments(
+    privkey: str, save_info: Optional[bool], contracts_version: Optional[str]
+) -> List:
     if save_info is None:
         save_info_arguments: List = []
     elif save_info is True:
         save_info_arguments = ["--save-info"]
     else:
         save_info_arguments = ["--no-save-info"]
-    common_arguments: List = [
-        "--private-key",
-        privkey,
-        "--max-token-networks",
-        1,
-        "--rpc-provider",
-        "rpc_provider",
-    ]
-    return common_arguments + save_info_arguments
+
+    if contract_version_with_max_token_networks(contracts_version):
+        max_arguments: List = ["--max-token-networks", 1]
+    else:
+        max_arguments = []
+
+    if contracts_version is None:
+        contracts_version_arguments: List = []
+    else:
+        contracts_version_arguments = ["--contracts-version", contracts_version]
+
+    common_arguments: List = ["--private-key", privkey, "--rpc-provider", "rpc_provider"]
+    return common_arguments + save_info_arguments + contracts_version_arguments + max_arguments
```

Comment:
Wouldn't it be easier to have only one `arguments` list append to conditionally? You could skip all the `..._arguments = []` branches and the concatenation at the end.

Reply:
I agree.  Will do in this PR.